### ISSUE TITLE
Incorrect arguments to removeEventListener

### ISSUE THREAD

**Issue by rtibbles** [learningequality/kolibri-design-system#398]
## Product
Observed in the destroy hook of `LibraryPage/index.vue` in the Kolibri learn plugin.

## Expected behavior
Event listeners should be properly removed.

## Actual behavior
This error was observed:
```
vue.runtime.esm.js:1897 TypeError: Failed to execute 'removeEventListener' on 'EventTarget': 2 arguments required, but only 1 present.
    at removeWindowListener (useKWindowDimensions.js:62:1)
    at useKWindowDimensions.js:81:1
    at VueComponent.<anonymous> (vue-composition-api.esm.js:883:1)
    at invokeWithErrorHandling (vue.runtime.esm.js:1863:1)
    at callHook (vue.runtime.esm.js:4235:1)
    at VueComponent.Vue.$destroy (vue.runtime.esm.js:4018:1)
    at destroy (vue.runtime.esm.js:3178:1)
    at invokeDestroyHook (vue.runtime.esm.js:6158:1)
    at removeVnodes (vue.runtime.esm.js:6174:1)
    at updateChildren (vue.runtime.esm.js:6279:1)
```

This seems to be the result of https://github.com/learningequality/kolibri-design-system/blob/develop/lib/useKWindowDimensions.js#L62 not specifying the event type, only the handler. At least the type and the handler must be specified: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener

Adding `'resize'` as the first argument would seem to be sufficient.

**Comment by pkspyder007** [comment]
@rtibbles I can try fixing this issue, can you assign it to me

**Comment by rtibbles** [comment]
Sure - please work from the develop branch!

**Comment by rtibbles** [comment]
Fixed in #400 by @pkspyder007
